src/core/public/i18n/i18n_service.tsx

```diff
@@ -35,25 +35,34 @@ export class I18nService {
       }),
     };
 
-    return {
-      /**
-       * I18n context is required by any localizable React component from @kbn/i18n and @elastic/eui packages
-       * and is supposed to be used as the topmost component for any i18n-compatible React tree.
-       */
-      Context: function I18nContext({ children }: { children: React.ReactNode }) {
+    const setup: I18nSetup = {
+      Context: function I18nContext({ children }) {
         return (
           <I18nProvider>
             <EuiContext i18n={{ mapping }}>{children}</EuiContext>
           </I18nProvider>
         );
       },
     };
+
+    return setup;
   }
 
   public stop() {
     // nothing to do here currently
   }
 }
 
-/** @public */
-export type I18nSetup = ReturnType<I18nService['setup']>;
+/**
+ * I18nSetup.Context is required by any localizable React component from \@kbn/i18n and \@elastic/eui packages
```

Comment:
Can I ask why moving this comment now has forward slashes in front of `@kbn` and `@elastic`?

Reply:
The TSDoc standard uses `@...` for tags, so anytime an '@' is used it needs to be escaped. Before these comments weren't exported as part of the type system, so they weren't interpreted by TSDoc/api-exporter, so it wasn't an issue.